packages/did-core-test-server/suites/did-spec/did-jsonld-consumption.js

```diff
@@ -0,0 +1,29 @@
+const deepEqual = require('deep-equal')
+
+const generateJsonldConsumptionTests = ({did, resolutionResult}) => {
+  const {didDocument} = resolutionResult;
+
+  it('6.3.2 JSON-LD Consumption - The DID document and any DID document ' +
+    'data structures expressed by a JSON-LD representation MUST be ' +
+    'deserialized into the data model according to the JSON ' +
+    'representation consumption rules as defined in § 6.2 JSON.', async () => {
+      reserializedDidDocument = JSON.parse(JSON.stringify(didDocument));
```

Comment:
almost like JSON-LD is JSON.

Reply:
Yes, all JSON-LD is JSON, but not all JSON is JSON-LD, so yes, differentiation is needed, and there will be a lot of overlap. So it goes.

Reply:
@OR13 wrote:
> almost like JSON-LD is JSON.

I'm as shocked as you are.